### PR TITLE
A* pathfinding

### PR DESCRIPTION
**Pull request by LeoWikstrom**

### CODE REVIEW

**Review by Schtekt — DISMISSED** [review]
woooho

File: StortSpelProjekt/Engine/src/Headers/EngineMath.h [hunk 1]
```diff
@@ -117,7 +123,7 @@ typedef union double3
 	void normalize()
 	{
 		double length = this->length();
-		if (length > 0.001)
+		if (length > EPSILON)
```
Comment (Anteanten) [inline StortSpelProjekt/Engine/src/Headers/EngineMath.h:126]: Tack

**Review by AntonAsbrink — CHANGES_REQUESTED** [review]
timScene krashar

**Review by Schtekt — APPROVED** [review]
All scenes seem to work now.

**Review by AntonAsbrink — APPROVED** [review]

**Pull request merged**